Story Update

kittykalYT opened the pull request:
finished writing entire story, now has 3 winning endings a whole lot of losing endings

kittykalYT committed:
Story Update

finished writing entire story, now has 3 winning endings a whole lot of losing endings

badiozam commented on kamrons-game.py:
```diff
@@ -42,33 +42,99 @@ def get_answer(prompt):
+                                        
                         else:
-                            print("You did nothing and the FBI ran over you. Game over.")
+                            print("Invalid command. You did nothing and the FBI ran over you. Game over.")
```
Instead of `Invalid command`, maybe something like `It seems like you are confused because that wasn't an option.`

kittykalYT replied:
on it, good idea

kittykalYT committed:
Update to Story-update

changed invalid command prompt to "looks like you were confused."

badiozam commented on kamrons-game.py:
```diff
@@ -42,33 +42,99 @@ def get_answer(prompt):
         
             else:
-                print("Invalid choice, you just stood there like an idiot as the FBI punched you to death.")
+                print("Looks like you were confused. you just stood there like an idiot as the FBI punched you to death.")
```
Capitalize `you`

badiozam replied:
Still needs capitalization.

kittykalYT replied:
OH there was TWOOOO

kittykalYT replied:
more than that actualy

badiozam commented on kamrons-game.py:
```diff
@@ -42,33 +42,99 @@ def get_answer(prompt):
     
         else:
-            print("Invalid choice, you ran straight into a wall in front of you, the FBI caught up to you, and you were never seen again.")
+            print("Looks like you were confused. you ran straight into a wall in front of you, the FBI caught up to you, and you were never seen again.")
```
Capitalize `you`

kittykalYT replied:
ok

kittykalYT committed:
Update 2 to Story-update

fixed punctuation error on line 137, "you" in "You ran straight into..." was not capitalized.

kittykalYT committed:
Update3 of Story-update

fixed even MORE punctuation errors that i did not see at all
